templates.js

```diff
@@ -154,6 +154,7 @@ window._paq.push(['trackEvent', 'UX Research', 'Excessive Scroll', lastKnownPosi
     {
       id: 'heatmapClicks',
       name: 'Heatmap clicks collector',
+      beforeDescriptionNote: '<b>IMPORTANT: If your PPAS version is higher than 16.8.0 you should migrate to integrated, backward compatible solution which is <a target="_blank" href="https://help.piwik.pro/support/tag-manager/heatmap-clickmap-tag/">Heatmap & clickmap tag</a></b>',
```

Comment:
I'm not sure whether we should recommend the migration. For sure we should recommend using the tag template for new setups.

Reply:
My purpose was only to engage people to use new tag solution if they find that code first. And in the end we should deprecate that code to not maintain them both.